src/workflows/bidsSpatialPrepro.m

```diff
@@ -44,12 +44,17 @@ function bidsSpatialPrepro(opt)
 
   [BIDS, opt, group] = setUpWorkflow(opt, 'spatial preprocessing');
 
+  opt.orderBatches.selectAnat = 1;
+  opt.orderBatches.realign = 2;
+  opt.orderBatches.coregister = 3;
+  opt.orderBatches.segment = 5;
```

Comment:
```suggestion
  opt.orderBatches.saveCoregistrationMatrix = 4;
  opt.orderBatches.segment = 5;
  opt.orderBatches.skullStripping = 6;
  opt.orderBatches.skullStrippingMask = 7;
```

Reply:
Technically we should also add all the different steps for the normalization and reslicing that can happen in that pipeline, but I am tempted to keep that for later because those steps only happen in some cases (see line 86 and 94).

The good news about "documenting" all the steps of the workflow like this is that it will make it much easier to auto-generate the method section that describes what the pipeline does.

Reply:
I think we should soon think of even having a simple function whose only goal will be to return the order of those steps for a certain workflow.